reproductions/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "uniforms-reproductions",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "parcel index.html",
+    "build": "parcel build index.html",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Let's clean it up a little:
* Add a `"private": true` to prevent accidental publish.
* Neither `version`, `description`, `main`, `scripts.test`, `keywords`, `author` nor `license` fields are needed as we're not going to publish anyway.